app/src/main/java/com/sun/dailynote/data/model/TodoChecks.java

```diff
@@ -0,0 +1,4 @@
+package com.sun.dailynote.data.model;
+
+public class TodoChecks {
```

Comment:
```suggestion
public class Task {
```
nên đặt là task or checklist thì ok hơn